src/less/widgets/mwstew.ui.PageHeadSectionWidget.less

```diff
@@ -17,8 +17,8 @@
 
 	&-title {
 		position: relative;
-		top: ~"calc(" @pageHeadSectionHeight / 2 ~" - 4em)";
-		text-shadow: rgba(0, 0, 0, 0.6) 0 1px 2px;
+		top: ~'calc(' @pageHeadSectionHeight / 2 ~' - 4em)';
```

Comment:
BTW I'm not sure what you're trying to achieve here w.r.t. to vertical alignment. Pretty sure you can do it without 'calc' though.